infra/deployer-service.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+find ./deployer-service ./lib ./config -name "*.go" | entr -r sh -c 'echo "Restarted deployer-service" && go run ./deployer-service'
```

Comment:
Hey @meiazero, sorry if I meant it wrong. It should be a single Makefile with all the scripts to run a given service. So a make script to run docker cmopose, another to run uploader,deployer...

and then a final dev that run them all. So no need to create `.sh` files and a infra folder

Reply:
It should be atomic, for the deployer: `go run ./deployer-service`

Reply:
you can add a separate called watch that will execute a second script instead of the `go run` bit

Reply:
> It should be atomic, for the deployer: `go run ./deployer-service`

Don't you watch the changes on the `deployer`?

Reply:
> you can add a separate called watch that will execute a second script instead of the `go run` bit

like this:

```make
watch:
	@find . -name "*.go" | entr -r sh -c 'echo "Restarted"
```

Reply:
After a few dozen tries, it seems that this script works satisfactorily.

```make
.PHONY: dev

compose:
	@docker-compose up -d || docker compose up -d

deployer:
	@make watch ; go run ./deployer-service

uploader:
	@make watch ; go run ./uploader-service

watch:
	@find . -name "*.go" | entr -r sh -c 'echo "Restarted"'

dev:
	@make compose ; make deployer ; make uploader

```